Assets/Plugins/Source/Editor/Utility/GUIEditorUtility.cs

```diff
@@ -993,6 +993,7 @@ private static void RenderSandboxInputs(ref ProductionEnvironments value)
                     if (!productionEnvironmentsCopy.Sandboxes.Remove(item))
                     {
                         // TODO: Tell user why the sandbox could not be removed.
+                        Debug.LogError($"{nameof(GUIEditorUtility)} {nameof(RenderSandboxInputs)}: Could not remove sandbox value .");
```

Comment:
Updating the message should cover this TODO

**(untested code)**
```suggestion
                        Debug.LogError($"{nameof(GUIEditorUtility)} {nameof(RenderSandboxInputs)}: Failed to find sandbox with name {item.Name} when trying to remove it.");
```

Reply:
This has been implemented in the following commit.